### PR TITLE
[WP #42101] Sort linked workpackages

### PR DESCRIPTION
**Pull request by nabim777**
## Description

The workpackages were previously sorted by their status, but in this PR there is implementation done for  sorting workpackages by the index in ascending order

## Related Issue or Workpackage

- Fixes [issue_link](https://community.openproject.org/projects/nextcloud-integration/work_packages/52780/activity)

## Screenshots (if appropriate):

| Before(sorted by workpackages status)   | After(sort by workpackages index)  |
|---|---|
|   |   |

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Updated `CHANGELOG.md` file

### CODE REVIEW

File: src/views/ProjectsTab.vue [hunk 1]
```diff
@@ -118,7 +118,7 @@ export default {
 					return false
 				}
 				return wp.fileId === this.fileInfo.id
-			})
+			}).sort((a, b) => a.id - b.id)
```
Comment (SagarGi) [inline src/views/ProjectsTab.vue:121]: what is `a` and `b` give some good name?

**Comment by SagarGi** [comment]
@nabim777  can you also add some unit tests for the function.

**Comment by nabim777** [comment]
As discussed there will be a change in the API to support fetching file link collections, which are then sortable on file link attributes like creation date on `openproject` side. So this PR is blocked till

**Comment by nabim777** [comment]
As per Dominic's comment in work packages https://community.openproject.org/projects/nextcloud-integration/work_packages/52780/activity#activity-9 The work package is rejected. We leave sorting as it is currently, So closing this issue for now

**Pull request closed without merging**